modules/test/playwright/fixtures/featureFlagsTest.ts

```diff
@@ -173,20 +169,22 @@ async function invokeServer<T>(
 					}),
 					method: 'POST',
 				})
-					.then((response) => response.text())
-					.then((text) => {
+					.then(async (response) => {
+						if (!response?.ok) {
+							reject(
+								`Feature Flag ${Liferay.Util.escapeHTML(
+									partialBody['key']
+								)} is not available`
+							);
+						}
```

Comment:
We should have two messages: one for 404 code, and one for 500 code. `!response?.ok` handles both.

Reply:
I dont know if its valuable to have a specific message for HTTP 500 :thinking: 
It would fall into catch block from Fetch API, so it will just reject the promise. Does it make sense? cc: @ethib137

Reply:
Actually, it doesn't fall on the catch block. From the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#differences_from_jquery.ajax):

> The promise returned from fetch() won't reject on HTTP errors even if the response is an HTTP 404 or 500. Instead, as soon as the server responds with headers, the promise will resolve (with the [ok](https://developer.mozilla.org/en-US/docs/Web/API/Response/ok) property of the response set to false if the response isn't in the range 200–299). The promise will only reject on network failure or if anything prevented the request from completing. 

I think it wouldn't be an issue to just add the message `an unexpected error has occurred` or something similar, this endpoint is for internal usage.

Reply:
If a 500 error takes place, I think there should be an error message shown. I'm not too concerned with what that message is since it wasn't a part of the original requirements, so you guys can make that decision.